dev/PushNotifications/MockBackgroundTaskInstance.h

```diff
@@ -1,34 +1,36 @@
-// Copyright (c) Microsoft Corporation and Contributors.
+﻿// Copyright (c) Microsoft Corporation and Contributors.
 // Licensed under the MIT License.
```

Comment:
Can you add a description in this format:

Motivation:

What changed:

Validation:

Makes it easy to understand why it is introduced in the first place?

Reply:
Yeah I can rephrase it better